webapp/models.py

```diff
@@ -44,4 +44,6 @@ class LogEntry(models.Model):
     habit = models.ForeignKey(
         Habit,
         on_delete = models.CASCADE,
+        null=False,
+        blank=False
```

Comment:
oh and since you are making a change to the model, make sure to makemigrations and add that to this commit

Reply:
yup, did that

Reply:
hmm I think you did it locally but maybe didn't add the new migration file to this commit cause I'm not seeing it in the files changed tab

Reply:
I did git push after changing the codes. Let me figure out how to get it up here

Reply:
should be another file in this folder: https://github.com/habitfarm/habitfarm/tree/master/webapp/migrations

Reply:
if you do `git status` on this branch, do you see one untracked file for the migration? If so, you can `git add` the file, then commit, then push and it should pop up here. Also, did you mean to close this PR?